### ISSUE TITLE
Update tax dialog screenshot in the manual

### ISSUE THREAD

**Issue by lmoureaux** [longturn/freeciv21#2068]
There is a new multi slider widget that looks different.

> I added an issue to capture the code for theme support. I do think we should either update this PR with the new images for the client manual or do it very soon in another PR.

_Originally posted by @jwrober in https://github.com/longturn/freeciv21/issues/2057#issuecomment-1828803801_

To do once theming is done or if it doesn't make it to 3.1

**Comment by lmoureaux** [comment]
Depends on #2067
